packages/payment-integrations-test-utils/src/test-utils/payment-integration-service.mock.ts

```diff
@@ -73,6 +74,7 @@ const PaymentIntegrationServiceMock = jest
             selectShippingOption,
             applyStoreCredit,
             verifyCheckoutSpamProtection,
+            updateProviderCustomerData,
```

Comment:
Can we call it `updatePaymentProviderCustomerInformation` ? This does seem a bit verbose though 🤔

Reply:
Or can just call it `updatePaymentProviderCustomer`

Reply:
`updatePaymentProviderCustomer` looks good to me